### PR TITLE
Add Sei Mainnet

### PR DESCRIPTION
**Pull request by dylanschultzie**
Signed-off-by: Dylan Schultz <hello@schultzie.dev>

### CODE REVIEW

**Review by dynalz — APPROVED** [review]

File: sei/chain.json [hunk 1]
```diff
@@ -0,0 +1,118 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "sei",
+    "status": "live",
+    "website": "https://www.sei.io/",
+    "network_type": "mainnet",
+    "pretty_name": "Sei",
+    "chain_id": "pacific-1",
+    "bech32_prefix": "sei",
```
Comment (JeremyParish69) [inline sei/chain.json:9]: Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Comment (dylanschultzie) [reply sei/chain.json:9]: Ugh I always forget that part. Thank you. 

https://github.com/satoshilabs/slips/pull/1584

File: sei/assetlist.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+            ],
+            "base": "usei",
+            "name": "Sei",
+            "display": "sei",
+            "symbol": "SEI",
+            "logo_URIs": {
+                "png": "",
+                "svg": ""
+            },
+            "coingecko_id": "sei-network"
```
Comment (udpatil) [inline sei/assetlist.json:25]: TODO: change this once it gets changed from coingecko's side

File: sei/assetlist.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                    "denom": "sei",
+                    "exponent": 6
+                }
+            ],
+            "base": "usei",
+            "name": "Sei",
+            "display": "sei",
+            "symbol": "SEI",
+            "logo_URIs": {
+                "png": "",
```
Comment (udpatil) [inline sei/assetlist.json]: TODO: update this

File: sei/chain.json [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    "node_home": "$HOME/.sei",
+    "key_algos": [
+        "secp256k1"
+    ],
+    "slip44": 118,
+    "fees": {
+        "fee_tokens": [
+            {
+                "denom": "usei",
+                "fixed_min_gas_price": 0,
```
Comment (udpatil) [inline sei/chain.json:20]: pacific-1 (sei mainnet) has a global min gas fee of `0.1` as of a recent gov prop. Additionally, if this value changes up or down in the future, we should update this via PR, right?

Comment (udpatil) [reply sei/chain.json:20]: Is this mean to serve as a source of truth for gas prices, or just a guidance? curious how strictly these values need to match the actual chain parameters

File: sei/chain.json [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+    "staking": {
+        "staking_tokens": [
+            {
+                "denom": "usei"
+            }
+        ]
+    },
+    "codebase": {
+        "git_repo": "https://github.com/sei-protocol/sei-chain",
+        "recommended_version": "v3.0.0",
```
Comment (udpatil) [inline sei/chain.json]: the sei release versions dont have `v` prefixing, and the recommended / latest version is `3.0.8`

https://github.com/sei-protocol/sei-chain/releases/tag/3.0.8

Comment (udpatil) [reply sei/chain.json]: we may make a change in the future to have `v` prefixing for best practice, but as of now that isn't the format

File: sei/chain.json [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+        "compatible_versions": [
+            "v3.0.0"
+        ],
+        "cosmos_sdk_version": "",
+        "cosmwasm_enabled": true,
+        "cosmwasm_path": "$HOME/.sei/wasm",
+        "genesis": {
+            "genesis_url": ""
```
Comment (udpatil) [inline sei/chain.json]:
```suggestion
            "genesis_url": "https://github.com/sei-protocol/testnet/blob/main/pacific-1/genesis.json"
```

Comment (udpatil) [reply sei/chain.json]: note: we likely will need to rename the repo from the original name of `testnet` to better reflect that it has mainnet stuff too, will probably name it like `networks` or something, but @udpatil to follow up on that

File: sei/chain.json [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+        "seeds": [
+            {
+                "id": "20e1000e88125698264454a884812746c2eb4807",
+                "address": "seeds.lavenderfive.com:11956",
+                "provider": "Lavender.Five Nodes 🐝"
+            }
+        ],
+        "persistent_peers": [
+            {
+                "id": "20e1000e88125698264454a884812746c2eb4807",
```
Comment (udpatil) [inline sei/chain.json:77]: what's the recommended practice for this, is the general best practice to have active set vals here? or just a handful of peers?

Comment (dylanschultzie) [reply sei/chain.json:77]: I actually recommend zero peers, as seeds are far superior, but inevitably people will add them, so I throw one in there to start the process.

File: sei/chain.json [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+                "provider": "Lavender.Five Nodes 🐝"
+            }
+        ]
+    },
+    "apis": {
+        "rpc": [
+            {
+                "address": "https://sei-rpc.lavenderfive.com:443",
+                "provider": "Lavender.Five Nodes 🐝"
+            }
```
Comment (udpatil) [inline sei/chain.json:88]:
```suggestion
            },
            {
                "address": "https://sei-rpc.polkachu.com/",
                "provider": "polkachu.com"
            },
            {
                "address": "https://sei-rpc.brocha.in/",
                "provider": "Brochain"
            },
            {
                "address": "https://rpc-sei.stingray.plus/",
                "provider": "StingRay"
            },
            {
                "address": "https://rpc-sei.rhinostake.com",
                "provider": "RHINO"
            }
```

File: sei/chain.json [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+        ],
+        "rest": [
+            {
+                "address": "https://sei-api.lavenderfive.com:443",
+                "provider": "Lavender.Five Nodes 🐝"
+            }
```
Comment (udpatil) [inline sei/chain.json:94]:
```suggestion
                "provider": "Lavender.Five Nodes 🐝"
            },
            {
                "address": "https://sei-api.polkachu.com/",
                "provider": "polkachu.com"
            },
             {
                "address": "https://sei-rest.brocha.in/",
                "provider": "Brochain"
            },
             {
                "address": "https://api-sei.stingray.plus/",
                "provider": "StingRay"
            },
             {
                "address": "https://rest-sei.rhinostake.com",
                "provider": "RHINO"
            }
```

File: sei/chain.json [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+        ],
+        "grpc": [
+            {
+                "address": "https://sei-grpc.lavenderfive.com:443",
+                "provider": "Lavender.Five Nodes 🐝"
```
Comment (udpatil) [inline sei/chain.json:99]:
```suggestion
                "provider": "Lavender.Five Nodes 🐝"
            },
            {
                "address": "https://sei-grpc.polkachu.com:11990/",
                "provider": "polkachu.com"
            }
```

(waiting on getting grpc port info for the other providers)

File: sei/chain.json [hunk 11]
```diff
@@ -0,0 +1,122 @@
...
+        ],
+        "rest": [
+            {
+                "address": "https://sei-api.lavenderfive.com:443",
+                "provider": "Lavender.Five Nodes 🐝"
+            }
+        ],
+        "grpc": [
+            {
+                "address": "https://sei-grpc.lavenderfive.com:443",
```
Comment (udpatil) [inline sei/chain.json:98]: @dylanschultzie is this correct? when i test with grpcurl it fails

Comment (dylanschultzie) [reply sei/chain.json:130]: Yep!

Screenshot 2023-08-15 at 8 09 24 AM

File: sei/chain.json (hunk 10 above)
Comment (dylanschultzie) [reply sei/chain.json:131]: @udpatil if they don't provide a port, it's generally 443.

**Pull request merged**